src/main/java/org/dinosaur/foodbowl/domain/store/api/StoreController.java

```diff
@@ -0,0 +1,40 @@
+package org.dinosaur.foodbowl.domain.store.api;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.store.application.StoreService;
+import org.dinosaur.foodbowl.domain.store.dto.StoreRequest;
+import org.dinosaur.foodbowl.domain.store.dto.StoreResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Validated
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/stores")
+@RestController
+public class StoreController {
+
+    private static final String ADDRESS_REQUEST_DELIMITER = "+";
+    private static final String ADDRESS_DELIMITER = " ";
+
+    private final StoreService storeService;
+
+    @GetMapping
+    public ResponseEntity<StoreResponse> findStore(@NotBlank(message = "주소는 반드시 포함되어야 합니다.") @RequestParam String address) {
```

Comment:
```suggestion
    public ResponseEntity<StoreResponse> findStore(@RequestParam @NotBlank(message = "주소는 반드시 포함되어야 합니다.")  String address) {
```
`@RequestParam`이 앞으로 오면 용도를 더 확실히 확인할 수 있을 것 같아요 :) 추가적으로 DB의 길이 제약 조건도 추가해주는건 어떨까요?

Reply:
`@RequestParam`은 앞으로 오도록 수정했습니다 ~
현재 API가 요청으로 부터 온 주소에 해당하는 **가게가 존재하는지 여부를 판단하는 기능**이라서, DB의 길이 제약 조건은 추가해주지 않아도 될 것 같아요 ! 어떻게 생각하시나요 ~

Reply:
말씀하신대로 등록이나 수정은 필요할 것 같지만, 조회는 필요없을 것 같네요 좋습니다 👍